src/fortanix/Logo/Logo.tsx

```diff
@@ -27,13 +28,13 @@ export const Logo = ({ subtitle, subtitleTrademark, ...propsRest }: LogoProps) =
         propsRest.className,
       )}
     >
-      <div className="_logo">
-        <img alt={`Fortanix ${subtitle}`} src={logoImage} className="_icon"/>
+      <div className={cx('_logo', !showSubtitle && 'no-subtitle')}>
```

Comment:
We don't need the `no-subtitle` class here, instead in CSS you can use `:has(:global(._subtitle))` to determine  if the subtitle is present.